### ISSUE TITLE
Package repository locations for 6.7.0 appears to be incorrect

### ISSUE THREAD

**Issue by krisATelastic** [elastic/elasticsearch#40497]
**Elasticsearch version** (`bin/elasticsearch --version`): `6.7.0`

**Plugins installed**: []

**JVM version** (`java -version`):

**Description of the problem including expected versus actual behavior**: In the documentation the APT/YUM repository locations look to be incorrect resulting in a 404.

**Steps to reproduce**:

 1. Goto the [6.6 repository page](https://www.elastic.co/guide/en/elasticsearch/reference/6.6/deb.html#deb-repo) as an example.
 2. Look at the repository definition URL --> https://artifacts.elastic.co/packages/6.x/apt
 3. Goto the [6.7 repository page](https://www.elastic.co/guide/en/elasticsearch/reference/6.7/deb.html#deb-repo)
4. check the same repository definition URL. This does not exist and sends back an error to the APT/YUM client, see below.

**Provide logs (if relevant)**:
```
Err:7 https://artifacts.elastic.co/packages/6.7/apt stable Release
  404  Not Found [IP: 151.101.194.222 443]
Reading package lists... Done
E: The repository 'https://artifacts.elastic.co/packages/6.7/apt stable Release' does not have a Release file.
```

**Comment by jasontedor** [comment]
This is a docs bug. It should be 6.x like 6.6. Thanks for the report, we will get it fixed.

**Comment by jasontedor** [comment]
Closed by #40507

**Comment by krisATelastic** [comment]
No problem @jasontedor 👍
